components/collapsible/collapsible.test.js

```diff
@@ -144,6 +153,16 @@ describe('Dialtone vue Collapsible Component Tests', function () {
     });
   });
 
+  describe('If anchor text and anchor slot content are falsy', function () {
+    it('should output error message', async function () {
+      const consoleErrorSpy = sinon.spy(console, 'error');
```

Comment:
You're setting a spy on the global `console.error` and not restoring it after. This is what is making other tests fail.